docs/hardware/receiver-selection.md

```diff
@@ -13,14 +13,17 @@ ExpressLRS is hashtag blessed with the benefit of there being many receivers ava
 | Jumper AION Mini RX | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3: | :checkered_flag:|
 | Vantac RX | :moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3: | 20dBm | :whale::whale: | :thumbsup_tone3: | :checkered_flag:|
 | BetaFPV Nano 2.4 | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | 20dBm | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
+| BetaFPV Lite 2.4 | :moneybag::moneybag: | :straight_ruler: | :thumbsup_tone3:  | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
```

Comment:
This has no LNA

| BetaFPV Lite 2.4 | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3:  | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |

Reply:
which is why there's a thumbsdown?

Reply:
You've got thumbsup in the LNA column